crates/tiny/config.yml

```diff
@@ -29,13 +34,25 @@ servers:
       # (useful when `pass` or `sasl` fields above are not used)
       # nickserv_ident: 'hunter2'
 
+      # Set /ignore for this server and all its tabs, default: false
+      # ignore: true  
+
+      # Set notifier for this server and all its tabs, default: mentions
+      # notifier: <off | mentions | messages>
```

Comment:
Syntax here is inconsistent with the `join` example above. Above we use `[off | mentions | messages]`, here `<off | mentions | messages>`. I don't have an opinion on which one makes more sense but we should pick one and be consistent. I wonder which one of these syntax is more common?

Reply:
I have seen `< blah | blah | blah >` used in `StructOpt`, so maybe that?